samples/snippets/pom.xml

```diff
@@ -41,9 +41,13 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-document-ai</artifactId>
+      <version>0.1.1</version>
```

Comment:
nit: check if this can be removed before merge in case it got added to bom in time.

Reply:
Still taking 0.1.0 in my local machine, do not know if it will pick latest version in kokoro. I will create a github issue associated with this problem.